### PR TITLE
[fpv] formal compile fix required by VC Formal

### PR DESCRIPTION
**Pull request by cindychip**
Thanks Shruti for reporting these issues. These are some compile errors
found by VC Formal run script.

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/formal/fpv_all [hunk 1]
```diff
@@ -18,34 +18,29 @@ done
...
   "rv_timer"
+  "spi_device"
   "uart"
-  "hmac"
-  "flash_ctrl"
-  "otp_ctrl"
-  "pwrmgr"
-  "usbuart"
   "usbdev"
-  "usb_fs_nb_pe"
```
Comment (msfschaffner) [inline hw/formal/fpv_all:33]: what happened to this guy? does this currently not compile?

Comment (cindychip) [reply hw/formal/fpv_all:33]: This one has two assertions and they are imported into usbdev. It does not have a FPV tb either. So I think testing it in usbdev is enough.

Comment (msfschaffner) [reply hw/formal/fpv_all:33]: ah ok then this should be fine, yes.

Comment (cindychip) [reply hw/formal/fpv_all:33]: Thanks michael.

**Pull request merged**